jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -320,13 +324,16 @@ async def stage_3_process(app: Flask, qsm: QueueService):
             batch_processing.business_identifier,
             InvoluntaryDissolutionService.EligibilityFilters(exclude_in_dissolution=False)
         )
+        if batch_processing.meta_data is None:
```

Comment:
Similarly, it's not the right place...

Reply:
updated

Reply:
still incorrect as per comment above for `stage_2_date`

Reply:
Updated